src/main/java/backend/resource/TurboIssue.java

```diff
@@ -391,6 +393,22 @@ public TurboIssue combineWithPullRequest(PullRequest pullRequest) {
         return getMatchedIssues(issues, query).stream().findFirst();
     }
 
+    /**
+     * @param queries
+     * @return number of matches to a list of query
+     */
+    private int getNumberOfMatchingQuery(List<String> queries) {
+        return Math.toIntExact(
+            queries.stream().filter(query -> Utility.containsIgnoreCase(getMatchableText(), query)).count());
+    }
+
+    /**
+     * @return string that can be matched by any query
```

Comment:
I think we can explain this a little more. This is like a string representation of issues that all queries are matched against.